opengrok-web/src/main/webapp/WEB-INF/tags/repository.tag

```diff
@@ -42,7 +41,7 @@ Portions Copyright (c) 2019, Krystof Tulinger <k.tulinger@seznam.cz>.
 
 <c:if test="${isSubrepository}">
     <c:set var="name"
-           value="${Paths.get(pageConfig.sourceRootPath).relativize(Paths.get(repositoryInfo.directoryName))}"/>
+           value="${pageConfig.getRelativePath(pageConfig.sourceRootPath, repositoryInfo.directoryName)}"/>
```

Comment:
you're not using the static call but of course it does not matter

Reply:
Good point.